PromoExchange/batman
# Remove extraneous favicons

https://trello.com/c/uCTJ5sHT
### Testplan
1. Start application
2. Navigate to mainsite, ensure promo exchange favicon is displayed
3. Navigate to a company store, ensure promo exchange favicon is displayed

🏁 
- [x] Rebase from development once rubocop has been merged.
  https://github.com/PromoExchange/batman/pull/671
